--- /usr/lib/GNUstep/SOGo/Templates/MailerUI/UIxMailFolderTemplate.wox    2020-07-31 15:43:31.987504204
+++ /usr/lib/GNUstep/SOGo/Templates/MailerUI/UIxMailFolderTemplate.wox    2020-07-31 15:43:32.918291298
@@ -139,9 +139,9 @@
         </md-input-container>
         <md-input-container flex="25">
           <md-select label:aria-label="Search scope">
-            <md-option value="subject" selected="selected"><var:string label:value="Subject"/></md-option>
+            <md-option value="subject"><var:string label:value="Subject"/></md-option>
             <md-option value="from"><var:string label:value="Sender"/></md-option>
-            <md-option value="subject_or_from"><var:string label:value="Subject or Sender"/></md-option>
+            <md-option value="subject_or_from" selected="selected"><var:string label:value="Subject or Sender"/></md-option>
             <md-option value="to_or_cc"><var:string label:value="To or Cc"/></md-option>
             <md-option value="body"><var:string label:value="Entire Message"/></md-option>
           </md-select>
